### DISCUSSION TITLE
Cloth Simulation as a dependency

### DISCUSSION THREAD

**Discussion by aeggler** [InteractiveComputerGraphics/PositionBasedDynamics#143] [Q&A]
I am trying to use the cloth simulator as a dependency. When including the header files in my code, the compiler finds them, but not the recursive ones. Adding PBD as a subdirectory did not help.  So what is the best way to include position-based dynamics as a dependency?

**Comment by janbender** [comment]
I use PBD as dependency of the project [SPlisHSPlasH](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH). I added PBD as external project in the CMake file. Maybe you can do the same in your project.
